generators/upgrade/index.js

```diff
@@ -204,21 +201,21 @@ module.exports = class extends BaseGenerator {
 
     _retrieveLatestVersion(npmPackage) {
         this.log(`Looking for latest ${npmPackage} version...`);
-        const commandPrefix = this.clientPackageManager === 'yarn' ? 'yarn info' : 'npm show';
+        const commandPrefix = 'npm show';
         const pkgInfo = shelljs.exec(`${commandPrefix} ${npmPackage} version`, { silent: this.silent });
```

Comment:
Use `lastest-version` package to fetch last version instead of `npm show`.
Related: https://github.com/jhipster/generator-jhipster/issues/11925#issuecomment-642677898

Reply:
not sure to understand what you want here